docs-source/content/userguide/managing-domains/domain-events.md

```diff
@@ -49,10 +54,10 @@ Each operator-generated event in a domain namespace contains the following field
  *  `reason`:  Short, machine understandable string that gives the reason for the transition to the object's current status.
  *  `message`:  String that describes the details of the event.
  *  `involvedObject`:  `V1ObjectReference` object that describes the Kubernetes resources with which this event is associated.
-    *  `name`:  String that describes the name of the resource, either a domain resource, in which case the name is the `domainUID`, or a namespace, in which case the name is the name of the namespace.
-    *  `namespace`:  String that describes the namespace of the event, which is the namespace of the domain resource for a domain event.
-    *  `kind`:  String that represents the REST resource this object represents. The value is `Domain` for a domain event, and `Namespace` for a namespace event.
-    *  `apiVersion`:  String that describes the `apiVersion` of the involved object, which is the `apiVersion` of the domain resource, for example, `weblogic.oracle/v8`, for a domain event, and is not set for a namespace event.
+    *  `name`:  String that describes the name of the resource that the event is associated with. It may be the `domainUID`, the name of the namespace that the operator watches, or the name of the operator pod.
+    *  `namespace`:  String that describes the namespace of the event, which is either the namespace of the domain resource, or the namespace of the operator.
+    *  `kind`:  String that represents the REST resource this object represents. The value is `Domain` for a domain event, `Namespace` for a namespace event in the domain namespace, or the `Pod` for the operator pod.
```

Comment:
represents the REST resource this object represents -> (not sure if this suggestion is accurate) represents the type of REST resource object
or the `Pod` for the operator pod. -> or `Pod` for the operator pod.

Reply:
Our description of 'kind' is verbose. The "REST resource" part seems superfluous. Kubernetes just uses the circular description as here: https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#required-fields.

Reply:
So, it should be, "the kind of object." Is there a reason why they don't use "type" (does it have a reserved meaning)?

Reply:
If my recollection is correct, the current text is from Kubernete doc, but somehow I am not able to find it anymore.
Instead, I came across the following:
"kind: a string that identifies the schema this object should have".

Reply:
In the latest commit, I changed this to "describes the kind of resource that this object represents". Let me know if this is okay.

Reply:
ok with me.